packages/ui-kit/src/components/Logger/LogProvider.tsx

```diff
@@ -0,0 +1,20 @@
+import React, { createContext } from 'react'
+
+import { LogLevel } from './Log.types'
+
+export type LogContextProps = {
+  logLevel: LogLevel
+}
+
+export const LogContext = createContext({ logLevel: LogLevel.Error })
+
+export type LogProviderProps = {
+  children?: React.ReactNode
+
+  /** The log level to use (defaults to "error"). */
+  logLevel?: LogLevel
+}
+
+export const LogProvider = ({ children, logLevel = LogLevel.Error }: LogProviderProps) => (
```

Comment:
What about defining a constant, DEFAULT_LOG_LEVEL?
```suggestion
const DEFAULT_LOG_LEVEL = LogLevel.Error

export const LogContext = createContext({ logLevel: DEFAULT_LOG_LEVEL })

export type LogProviderProps = {
  children?: React.ReactNode

  /** The log level to use (defaults to "error"). */
  logLevel?: LogLevel
}

export const LogProvider = ({ children, logLevel = DEFAULT_LOG_LEVEL }: LogProviderProps) => (
```